mirrord-layer/src/common.rs

```diff
@@ -29,3 +40,172 @@ pub enum HookMessage {
     File(HookMessageFile),
     GetAddrInfoHook(GetAddrInfoHook),
 }
+
+mod common_ops {
+    use std::{os::unix::prelude::RawFd, sync::OnceLock};
+
+    use libc::c_int;
+
```

Comment:
Why do you need this here? Can't you use the already global `ENABLED_FILE_OPS` we have?